### PR TITLE
Use kyma-gateway instead of compass-gateway during kyma installation

### PR DESCRIPTION
**Pull request by PetarTodorovv**
Changes proposed in this pull request:

Use kyma-gateway instead of compass-gateway during kyma installation

**Related issue(s)**
#1553

### CODE REVIEW

**Review by dzahariev — APPROVED** [review]

File: chart/compass/charts/gateway/templates/gateway-certs-mtls-cacert.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and (eq .Values.gateway.enabled true) (eq .Values.gateway.manageCerts true) }}
+{{- if .Values.gateway.manageCerts }}
```
Comment (DimitarPetrov) [inline chart/compass/charts/gateway/templates/gateway-certs-mtls-cacert.yaml:1]: change file name to cmp-root-ca.yaml

File: installation/resources/kyma/installer-overrides-api-gateway.yaml [hunk 2]
```diff
@@ -8,5 +8,5 @@ metadata:
     component: api-gateway
     kyma-project.io/installation: ""
 data:
-  tests.env.gatewayName: "compass-istio-gateway"
-  tests.env.gatewayNamespace: "compass-system"
+  tests.env.gatewayName: "kyma-gateway"
```
Comment (DimitarPetrov) [inline installation/resources/kyma/installer-overrides-api-gateway.yaml:11]: This are the same values as the defaults from kyma we can remove the whole file.

File: installation/resources/kyma/installer-overrides-compass-gateway.yaml [hunk 3]
```diff
@@ -7,6 +7,7 @@ metadata:
     installer: overrides
     kyma-project.io/installation: ""
 data:
-  global.istio.gateway.name: "compass-istio-gateway"
-  global.istio.gateway.namespace: "compass-system"
+  global.istio.gateway.name: "kyma-gateway"
```
Comment (DimitarPetrov) [inline installation/resources/kyma/installer-overrides-compass-gateway.yaml:10]: This are the same values as the defaults from kyma we can remove the whole file.

File: installation/resources/kyma/installer-overrides-tracing.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+kind: ConfigMap
+metadata:
+  name: tracing-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: tracing
+    kyma-project.io/installation: ""
+data:
+  global.tracing.enabled: "true"
```
Comment (DimitarPetrov) [inline installation/resources/kyma/installer-overrides-tracing.yaml:11]: This are the same values as the defaults from kyma we can remove the whole file.

File: installation/scripts/install-kyma.sh [hunk 5]
```diff
@@ -45,6 +45,8 @@ ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
 INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/kyma/installer-cr-kyma-dependencies.yaml
 OVERRIDES_COMPASS_GATEWAY="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-compass-gateway.yaml
 API_GATEWAY_OVERRIDES="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-api-gateway.yaml
+TRACING_OVERRIDES="${ROOT_PATH}"/installation/resources/kyma/installer-overrides-tracing.yaml
```
Comment (DimitarPetrov) [inline installation/scripts/install-kyma.sh:48]: Not needed if other comments are addressed.

**Review by DimitarPetrov — CHANGES_REQUESTED** [review]

File: installation/resources/kyma/installer-overrides-api-gateway.yaml (hunk 2 above)
Comment (PetarTodorovv) [reply installation/resources/kyma/installer-overrides-api-gateway.yaml:11]: The file is removed

File: installation/resources/kyma/installer-overrides-compass-gateway.yaml (hunk 3 above)
Comment (PetarTodorovv) [reply installation/resources/kyma/installer-overrides-compass-gateway.yaml:10]: The file is removed

File: installation/resources/kyma/installer-overrides-tracing.yaml (hunk 4 above)
Comment (PetarTodorovv) [reply installation/resources/kyma/installer-overrides-tracing.yaml:11]: There is a kyma overrides that change this value to "false" so we need to override it again to "true"

File: installation/scripts/install-kyma.sh (hunk 5 above)
Comment (PetarTodorovv) [reply installation/scripts/install-kyma.sh:48]: Not applicable

**Review by DimitarPetrov — APPROVED** [review]

**Review by dzahariev — APPROVED** [review]

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**